src/test/java/hudson/scm/SubversionSCMTest.java

```diff
@@ -61,7 +61,6 @@
 import org.tmatesoft.svn.core.wc.SVNCommitClient;
 import org.tmatesoft.svn.core.wc.SVNRevision;
 import org.tmatesoft.svn.core.wc.SVNStatus;
-import org.tmatesoft.svn.core.wc.SVNWCUtil;
```

Comment:
It adds unnecessary history to the diff. If I want to view the changes made for JENKINS-29340, I should only see the _necessary_ changes. The removal of an import in `SubversionSCMTest` has nothing to do with the changes for JENKINS-29340. We should create a new PR that does this type of cleanup. Comment is more about future pull requests.